ravem/indico_ravem/util.py

```diff
@@ -112,16 +107,28 @@ def has_access(event_vc_room, _split_re=re.compile(r'[\s,;]+')):
     event = event_vc_room.event
     current_user = session.user
 
-    # No physical room or room is not Vidyo capable
-    if not room or not room.has_equipment('Vidyo'):
+    # No physical room or room is not videoconference capable
+    feature = RavemPlugin.settings.get('room_feature')
+    equipment = EquipmentType.query.filter(EquipmentType.features.any(RoomFeature.name == feature)).all()
+    if not room or not room.has_equipment(*(e.name for e in equipment)):
```

Comment:
I think this can be simplified as:

`if not room or not (feature.equipment_types & room.available_equipment):`

Reply:
Unfortunately, it doesn't seem to work with `InstrumentedList'`

Reply:
`set()` around it should fix that ;)

Reply:
Yes it does, nice, I thought the contents of both were unhashable.